op-service/eth/blob.go

```diff
@@ -14,7 +14,16 @@ import (
 
 const (
 	BlobSize        = 4096 * 32
-	MaxBlobDataSize = 4096*31 - 4
+	MaxBlobDataSize = (4*31+3)*1024 - 4
+	EncodingVersion = 0
+	VersionOffset   = 1    // offset of the version byte in the blob encoding
+	Rounds          = 1024 // number of encode/decode rounds
+)
+
+var (
+	InvalidFieldElementError    = errors.New("invalid field element")
```

Comment:
nit: errors are usually prefixed with `Err` and we may want to prefix with `ErrBlob` because this is in the `eth` package

Reply:
added ErrBlob prefix